Linux.md

```diff
@@ -67,6 +67,19 @@
 ### <a name="findAFile"></a> Find a file in Linux
 `$ sudo find . -name <file_name>`
 
+
+### Check IP to whitelist
+`curl ifconfig.me ; echo`
+
+### Change the ownership of a file
+`sudo chown new_owner_name filename`
```

Comment:
Can you pls give one use-case in which scenario one will be compelled to use this command?